### PR TITLE
Switch fr.openfisca.org to HTTPS

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to openfisca/openfisca-core#572

Depends on https://github.com/openfisca/legislation-explorer/pull/122

Partially in prod : [https](https://fr.openfisca.org/api/v2/parameters) works but [http](http://fr.openfisca.org/api/v2/parameters) doesn't redirect towards HTTPS yet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,25 +17,36 @@ For more information on `auto-update` sub-directories, please see their own READ
...
 
+
+## Set up a SSL certificate
+
+See the [dedicated page](guides/Set-up-SSL.md).
+
 ## Renew SSL certificates
 
-To renew the SSL certificate of an OpenFisca related application, run in `root` the following command, replacing `api-test.openfisca.fr` by the domain that needs a certificate update:
+To renew the SSL certificate of an OpenFisca related application, run in `root` the following command, replacing `fr.openfisca.org` by the domain that needs a certificate update:
```
Comment (Anna-Livia) [inline README.md]: > Should we put a reminder somewhere of when the certificate expire ? (in the calendar ?)

File: guides/Set-up-SSL.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Set up HTTPS on a domain name
+
+To handle `HTTPS` on a website, we need an up-to-date SSL **certificate**. For `openfisca.fr` and `openfisca.org`, we use [Let’s Encrypt](https://letsencrypt.org/) as a certificate provider.
+
+Note that we need one certificate per subdomain:
+- `openfisca.org` and `fr.openfisca.org` use **two different** certificates.
+- `fr.openfisca.org` and `fr.openfisca.org/api/` use **the same** certificate, even if they are different applications.
```
Comment (Anna-Livia) [inline guides/Set-up-SSL.md]: 👍

File: guides/Set-up-SSL.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    location / {
+        ...
+    }
+
+} 
+```
+
+Then run `service nginx reload` to take your changes into account.
+
+At this point, `https://fr.openfisca.org` should work! Make sure it does before going further.
```
Comment (Anna-Livia) [inline guides/Set-up-SSL.md]: How should I do that ?

Comment (fpagnoux) [reply guides/Set-up-SSL.md]: Type https://fr.openfisca.org in your browser, (don't forget the _s_ at the end of _http_). If the page load and you don't get a certificate error, everything is working 🙂

File: README.md (hunk 1 above)
Comment (fpagnoux) [reply README.md]: It expires every 3 months, so I'd definitely not put it here on a repository.

We can put it in the calendars, but they are many certificates, and it's not that obvious to get all their expiration dates.

Normally, Uptime robot will soon warn us before a certificate expires (new feature released yesterday).
We should also consider using certificate auto-renewal : https://github.com/openfisca/openfisca-ops/issues/14

**Comment by fpagnoux** [comment]
I'm not sure what do with the review, as it's neither an "Accepted", nor a "Requested Change".

To avoid any ambiguity : 

- If a PR is **not** ready not merge as is, or if you can't endorse the code without some clarification before accepting it, use "Request changes".
- If a PR is mergeable as is, but you have non-blocking suggestions or feedback, use "Accept" while indicating what you sugget
- Only use "Comment" if you consider that you're unable to review, and that either you need someone else to review the code, or you need an IRL discussion about the code. If so, clearly express why you consider yourself unable to review.

File: guides/Set-up-SSL.md (hunk 3 above)
Comment (Anna-Livia) [reply guides/Set-up-SSL.md]: Can we put this in `guides/Set-up-SSL.md` ?

Comment (fpagnoux) [reply guides/Set-up-SSL.md]: 👌

**Pull request merged**